File: 1323.maximum-69-number.java
/*
 * @lc app=leetcode id=1323 lang=java
 *
 * [1323] Maximum 69 Number
 */

// @lc code=start
class Solution {
    public int maximum69Number (int num) {
        char[] idk = String.valueOf(num).toCharArray();
        for (int i = 0; i < idk.length; i++) {
            if (idk[i] == '6') {
                idk[i] = '9';
                break;
            }
        }
        return Integer.valueOf(new String(idk));
    }
}
// @lc code=end

File: 1323.maximum-69-number.py
#
# @lc app=leetcode id=1323 lang=python
#
# [1323] Maximum 69 Number
#

# @lc code=start
class Solution(object):
    def maximum69Number (self, num):
        """
        :type num: int
        :rtype: int
        """
        s = list(str(num))
        for i in range(len(s)):
            if s[i] == '6':
                s[i] = '9'
                break
        n = ''.join(s)
        return int(n)
        
# @lc code=end

